bubbles/message_utils.py

```diff
@@ -1,19 +1,74 @@
-from typing import Any, Dict, Union
+from __future__ import annotations
 
+from typing import Any, Dict, Union, Callable, Optional, TYPE_CHECKING
+from slack_sdk.web.client import WebClient
 
-class MessageUtils:
-    def __init__(self, client, payload):
+if TYPE_CHECKING:
+    from bubbles.plugins import PluginManager
+
+
+class Payload:
+    """Payload class for everything a command needs."""
+    def __init__(
+            self,
+            client: WebClient=None,
+            slack_payload: dict=None,
+            say: Callable=None,
+            context: dict=None,
+            meta: PluginManager=None,
+    ):
         self.client = client
-        self.payload = payload
+        self._payload = slack_payload
```

Comment:
Let's call it `self.slack_payload` to make the distinction clearer

Reply:
I want this to be a "private" variable - the idea is that if you need something out of it, write a function that can be found by your editor. How about _slack_payload?

Reply:
That works for me!